package day45_exceptions;

public class BadArrray {

	public static void main(String[] args) {
		
		int[] number = {1,2,3} ;
		
		for(int i=0 ; i<=number.length ; i++) {
			System.out.println(number[i]);
		}
		
		System.out.println("learning exception");

	}

}
